BackToBasicsUIKit/BackToBasicsUIKit/MainTableViewController.swift

```diff
@@ -66,11 +67,19 @@ class MainTableViewController: UITableViewController {
         case 7: demoVC = NilsonViewController()
         case 8: demoVC = PauloViewController()
         case 9: demoVC = RyanViewController()
+        case 10: demoVC = JimmyViewController()
+            
         default:
             assertionFailure("Cell not configured yet")
         }
-
+        
+        if indexPath.row == 10 {
+            let storyboard = UIStoryboard(name: "Jimmy", bundle: nil)
+            let destination = storyboard.instantiateViewController(withIdentifier: "JimmyViewController") as! JimmyViewController
```

Comment:
💂

Reply:
Instead of the if statement, you can attach the if statement action to the switch statement and then return, example:

`case 10: 
let storyboard = UIStoryboard...


return`

Reply:
Don't use a storyboard. Another important lesson to learn here is that you're working on a codebase with a team and you should conform to the style and architecture of it. Nobody else is using storyboards and you shouldn't either.
Be a nice team citizen and conform to the style or convince the team that we should change the style. Don't go rogue though.